sda-commons-starter-web/src/test/java/org/sdase/commons/spring/boot/web/security/handler/app/ValidationExceptionHandlerTestApp.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright 2022- SDA SE Open Industry Solutions (https://www.sda.se)
+ *
+ * Use of this source code is governed by an MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ */
+package org.sdase.commons.spring.boot.web.security.handler.app;
+
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication(scanBasePackages = "org.sdase.commons.spring.boot.web.security.handler.app")
```

Comment:
I'd prefer to use `scanBasePackageClasses` because it is more robust to changes.